modules/database/sqlite/deactivate.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Actions to run when the module gets deactivated.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Deletes the db.php file, and deactivates the module in the SQLite database.
+ *
+ * @since n.e.x.t
+ */
+return function() {
+	if ( ! file_exists( WP_CONTENT_DIR . '/db.php' ) ) {
+		return;
+	}
+
+	global $wp_filesystem;
+	if ( ! $wp_filesystem ) {
+		require_once ABSPATH . '/wp-admin/includes/file.php';
+		WP_Filesystem();
+	}
+	$wp_filesystem->delete( WP_CONTENT_DIR . '/db.php' );
```

Comment:
This is problematic, as it could also be another `db.php` than our own. We need to make sure this is _our_ `db.php` file before deleting it. For example, in the module activation routine we bail when an existing `db.php` is found. The module will still remain active though, so when the user then deactivates it the code here likely deletes a `db.php` file that is not ours, potentially breaking the website.

Reply:
Fixed when I added the `PERFLAB_SQLITE_DB_DROPIN_VERSION` constant 👍